packages/lib/src/core/Analytics/utils.ts

```diff
@@ -0,0 +1,30 @@
+import { AnalyticsObject } from './types';
+import { ANALYTICS_ACTION_STR, ANALYTICS_SUBMIT_STR } from './constants';
+
+export const getUTCTimestamp = () => Date.now();
+
+/**
+ * All objects for the /checkoutanalytics endpoint have base props:
+ *  "timestamp" & "component" (and an optional "metadata" object of key-value pairs)
+ *
+ * Error objects have, in addition to the base props:
+ *  "code", "errorType" & "message"
+ *
+ * Log objects have, in addition to the base props:
+ *  "type" & "target" (e.g. when onSubmit is called after a pay button click), or,
+ *  "type" & "subtype" & "message" (e.g. when an action is handled), or,
+ *  "type" & "message" (e.g. logging during the 3DS2 process)
+ *
+ * Event objects have, in addition to the base props:
+ *   "type" & "target"
+ */
+export const createAnalyticsObject = (aObj): AnalyticsObject => ({
```

Comment:
Is there any specific type for the `aObj` or should it be `any` in this case?

Reply:
Fixed